stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -265,6 +300,66 @@ public fun MessageComposer(
     }
 }
 
+/**
+ * Represents the default content shown at the top of the message composer component.
+ *
+ * @param messageComposerState The state of the message composer.
+ * @param onCancelAction Handler for the cancel button on Message actions, such as Edit and Reply.
+ */
+@Composable
+public fun DefaultMessageComposerHeaderContent(
+    messageComposerState: MessageComposerState,
+    onCancelAction: () -> Unit,
+) {
+    val activeAction = messageComposerState.action
+
+    if (activeAction != null) {
+        MessageInputOptions(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 8.dp, bottom = 6.dp, start = 8.dp, end = 8.dp),
+            activeAction = activeAction,
+            onCancelAction = onCancelAction
+        )
+    }
+}
+
+/**
+ * Represents the default content shown at the bottom of the message composer component.
+ *
+ * @param messageComposerState The state of the message composer.
+ * @param onShowInChannelChecked Handler when the user checks the also send to channel checkbox.
+ */
+@Composable
+public fun DefaultMessageComposerFooterContent(
+    messageComposerState: MessageComposerState,
+    onShowInChannelChecked: (Boolean) -> Unit,
+) {
+    if (messageComposerState.messageMode is MessageMode.MessageThread) {
```

Comment:
This is a nitpick and feel free to disregard ;)

For better readability, could we "guard" against this statement here? ```(messageComposerState.messageMode is MessageMode.MessageThread)```

Dunno wha'ts the most fancy way to do it in Kotlin, but here's the logic that I have on my mind:

```
if !(messageComposerState.messageMode is MessageMode.MessageThread) return
```

This way the reader no longer needs to account for this condition when reading the body of this function.

Reply:
In compose - it's actually a practice to declare when a UI component will be emitted, rather than when it's not

That way we know that in case a condition isn't met, it won't be called and we'll have this empty call

it's a bit different in swiftUI I guess because you do have objects and references, so you can always refer to stuff